add task HTML form solution

SergiyVS opened the pull request:
- [DEMO LINK](https://SergiyVS.github.io/layout_html-form/)
- [TEST REPORT LINK](https://SergiyVS.github.io/layout_html-form/report/html_report/)

aionova commented on src/index.html:
```diff
@@ -1,15 +1,210 @@
+              maxlength="20"
+              autocomplete="off"
+              class="input"
+              required>
```
```suggestion
              required
            >
```

aionova commented on src/index.html:
```diff
@@ -1,15 +1,210 @@
+             name="name"
+             id="name"
+             required
+             minlength="4"
+            autocomplete="off"
+            class="input">
```
```suggestion
             minlength="4"
             autocomplete="off"
             class="input"
           >
```

aionova commented on src/index.html:
```diff
@@ -1,15 +1,210 @@
+             min="5"
+             max="100"
+             autocomplete="off"
+             class="input">
```
```suggestion
             class="input"
           >
```

aionova commented on src/index.html:
```diff
@@ -1,15 +1,210 @@
+              type="checkbox"
+              name="agree"
+              id="agree"
+              required>
```
```suggestion
              required
            >
```

aionova replied:
same everywhere

aionova commented on src/index.html:
```diff
@@ -1,15 +1,210 @@
+        </div>
+
+        <div>
+          <label for="old">
```
```suggestion
          <label for="age">
```

aionova commented on src/index.html:
```diff
@@ -1,15 +1,210 @@
+            I accept the term of the agreement
+            <input
+              type="checkbox"
+              name="agree"
```
```suggestion
              name="agreeWithTerms"
```

aionova commented on src/index.html:
```diff
@@ -1,15 +1,210 @@
+            <input
+              type="radio"
+              name="cats_answer"
+              id="radio_cat_no">
```
don't indicate input type in id. It should describe content of the input, not its type

aionova commented on src/index.html:
```diff
@@ -1,15 +1,210 @@
+        </div>
+
+        <div>
+          <label for="time">
```
```suggestion
          <label for="time-to-bed">
```

aionova commented on src/index.html:
```diff
@@ -1,15 +1,210 @@
+        </div>
+
+        <div>
+          <label for="range">
```
same here. Don't include input type in id

aionova commented on src/index.html:
```diff
@@ -1,15 +1,210 @@
+        <div>
+          <label for="recomend">
+            Would you recommend us?
+            <select name="recomend" id="recomend">
```
```suggestion
            <select name="recommend" id="recommend">
```

aionova reviewed (changes requested):
Good job! Formatting and naming fixes left

aionova reviewed (approved):
Nice!

SergiyVS committed:
add task HTML form solution

Anastasiia-Svintsova commented on src/index.html:
```diff
@@ -11,7 +11,163 @@
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <!-- <h1>HTML Form</h1> -->
```
Remove all comments, please

Anastasiia-Svintsova commented on src/index.html:
```diff
@@ -11,7 +11,163 @@
-    <h1>HTML Form</h1>
+    <!-- <h1>HTML Form</h1> -->
+
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
```
Don't add spaces between parent and child components
```suggestion
    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
```

Anastasiia-Svintsova commented on src/index.html:
```diff
@@ -11,7 +11,163 @@
+          </select>
+        </label>
+
+      </fieldset>
```
Fix it everywhere
```suggestion
        </label>
      </fieldset>
```

Anastasiia-Svintsova reviewed (changes requested):
- Check labels for the radio-buttons 

https://user-images.githubusercontent.com/99403198/230059487-1bd2991c-59e4-4288-94cb-a56593aec4ed.mov

SergiyVS committed:
Re-request add task HTML form solution

Changed labels for the radio-buttons
Removed all comments
Deleted spaces between parent and child components

anstsot reviewed (approved):
Great job, approved!
